docs/getting-started.md

```diff
@@ -202,19 +202,47 @@ $ENV:AWS_SECRET_ACCESS_KEY=$awsCredentials.SecretAccessKey
 
 ## Deploying a cluster
 
+### Starting Minikube
+For security reasons, it's best to start a Minikube instance before using
+clusterctl to store your AWS credentials.
+
+``` shell
+minikube start
+```
+
+Then write the AWS credentials into the Minikube instance for use by Cluster API:
+
+``` shell
+minikube ssh 'mkdir -p .aws'
+clusterawsadm alpha bootstrap generate-aws-default-profile | minikube ssh 'cat > .aws/credentials'
+[default]
+aws_access_key_id = AKIAIOSFODNN7EXAMPLE
+aws_secret_access_key = wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+region = eu-west-1
+
+
+^C
+```
+
+**NOTE**:
+> This will print the AWS credentials to the screen and then hang
+> (due to a restriction in Minikube).
+> This is OK, and you can cancel with Ctrl+C, but make sure to do this privately.
+
 ### Generating cluster manifests
 
-The shell script `generate-yaml.sh` in [clusterctl/examples](clusterctl/examples) can be used to generate the cluster manifests.
+The shell script `config/generate-yaml.sh` can be used to generate the
+cluster manifests, and copy your AWS environment credentials into the
+minikube environment.
```

Comment:
Aren't the credentials already copied into the minikube env before running `generate-yaml.sh`?

Reply:
Thanks. Went through a few iterations and forgot to clean this up.